web/pdf_find_bar.js

```diff
@@ -88,6 +93,7 @@ var PDFFindBar = (function PDFFindBarClosure() {
       event.initCustomEvent('find' + type, true, true, {
         query: this.findField.value,
         caseSensitive: this.caseSensitive.checked,
+        phraseSearch: this.phraseSearch.checked,
```

Comment:
Again, here consider adding a null check. And if the item does not exist, default to true:
```
phraseSearch: this.phraseSearch ? this.phraseSearch.checked : true,
```